### PR TITLE
feat: [sc-15957] Implement degree.md in api documentation

### PR DESCRIPTION
**Pull request by AStoerkel1**
Story details: https://app.shortcut.com/clarkcan/story/15957

image

### CODE REVIEW

**Review by pmanal1 — APPROVED** [review]

File: docs/competency-api/degree.md [hunk 1]
```diff
@@ -4,4 +4,50 @@ title: Degree
 nav_order: 5
 parent: Competency API
 ---
-# Degree
\ No newline at end of file
+# Degree
+
+## Update Degree
+
+### Description
```
Comment (Cwagne17) [inline docs/competency-api/degree.md:11]: Needs a description

File: docs/competency-api/degree.md [hunk 2]
```diff
@@ -4,4 +4,50 @@ title: Degree
...
+| correct | false | string | What it means for this competency to be adequately correct |
+| time | false | string | How much time is allowed for the competency to be completed |
+| userId | true | Object Id: string | The userId matching the Bearer token |
+
+<details closed markdown="block">
+  <summary>
+    Example
+  </summary>
+```json
+{
```
Comment (Cwagne17) [inline docs/competency-api/degree.md:41]: Can we have a curl request example after postman is formatted.

**Review by Cwagne17 — CHANGES_REQUESTED** [review]

File: docs/competency-api/degree.md [hunk 3]
```diff
@@ -4,4 +4,50 @@ title: Degree
 nav_order: 5
 parent: Competency API
 ---
-# Degree
\ No newline at end of file
+# Degree
```
Comment (JosueMolinaMorales) [inline docs/competency-api/degree.md:7]: I noticed you don't have a note about headers here. Can you add that?

**Review by JosueMolinaMorales — COMMENTED** [review]
Looks good. Just chris's comments and something i left.

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by shanioffe — APPROVED** [review]
Looks good

File: docs/competency-api/degree.md [hunk 4]
```diff
@@ -4,4 +4,73 @@ title: Degree
...
+  -L "https://api.competency-constructor.clark.center/competencies/6112745b84804cf5833aa94c/degree"
+```
+</details>
+
+### Response
+
+| Name | Description |
+|:----:|----|
+| 200 | Degree updated successfully |
+| 400 | Competency is deprecated, rejected, or published and cannot be updated. Or body is incorrectly formatted. |
```
Comment (mfranz1) [inline docs/competency-api/degree.md]: Can it be made more obvious that this is a list of 400 errors

**Review by mfranz1 — CHANGES_REQUESTED** [review]

**Review by Cwagne17 — APPROVED** [review]

**Review by mfranz1 — APPROVED** [review]

**Pull request merged**